packages/ckb/src/utils/ckb-tx.ts

```diff
@@ -162,6 +162,11 @@ export const isSporeCapacitySufficient = (sporeCell: IndexerCell) => {
   return capacity - occupiedCapacity > BigInt(BTC_TIME_CELL_INCREASED_SIZE) * CKB_UNIT;
 };
 
+// If the spore cell capacity is insufficient, increase it by BTC_TIME_CELL_INCREASED_SIZE
+export const increaseSporeCapacity = (currentSporeCapacity: Hex): Hex => {
+  return `0x${(BigInt(currentSporeCapacity) + BigInt(BTC_TIME_CELL_INCREASED_SIZE) * CKB_UNIT).toString(16)}`;
```

Comment:
LGTM.

another way is `return calculateRgbppSporeCellCapacity(..., reserveMoreCkb = true)`

Reply:
They are different.

Assume the occupied capacity is 200CKB, the `calculateRgbppSporeCellCapacity` will return `200 + 1 + 95` and the `increaseSporeCapacity` will return `200.99876543 + 95`. 

`currentSporeCapacity` will be bigger than 200CKB and smaller than 201CKB(some CKB has already been used for transaction fee), I think the `increaseSporeCapacity` will be better.